proto/qdrant/cloud/cluster/backup/v1/backup.proto

```diff
@@ -137,12 +137,27 @@ message ListBackupsRequest {
   // The identifier of the schedule (in GUID format).
   // When this field is set, only backups triggered by the backup schedule are returned.
   optional string backup_schedule_id = 3 [(buf.validate.field).string = {uuid: true}];
+  // Maximum number of items to return.
+  // If not specified, all items are returned.
+  optional uint32 page_size = 10 [(buf.validate.field).uint32.gt = 0];
```

Comment:
would it make sense to add a value for `lt`, just for safety?

Reply:
Good idea, what value (<=250)?

Reply:
added and make it int32